packages/connect/README.md

```diff
@@ -1 +1,51 @@
+@0xproject/connect
+------
+
 This repository contains a Javascript library that makes it easy to interact with Relayers that conform to the [Standard Relayer API](https://github.com/0xProject/standard-relayer-api)
+
+## Installation
+
+```bash
+yarn add @0xproject/connect
+```
+
+## Usage
+
+* [Docs](https://0xproject.com/docs/connect)
+* [Tutorials](https://0xproject.com/wiki#connect)
+
+## Contributing
+
+We strongly recommend the community to help us make improvements and to determine the future direction of the protocol. To report bugs within this package, please create an issue in this repository.
+
+Please read our [contribution guidelines](../../CONTRIBUTING.md) before getting started.
+
+### Install Dependencies
+
+If you don't have yarn workspaces enabled - enable them:
```

Comment:
Let's add `(enabled by default starting at Yarn v1.0)`

Reply:
Or better yet: `If you don't have yarn workspaces enabled (Yarn < v1.0) - enable them:`